### ISSUE TITLE
[SCREENREADER]: Announce the number of results and any change in result listings to assistive devices

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#984]
## Description

Changes in the number of results, and which results are being shown are not announced to screen readers. This could be confusing because assistive device users are left to figure out we've loaded a new page of results manually. Screen shot attached below.

## Related Issues
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/981

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* As a screen reader user, I want to hear the number of results read aloud when I filter or click one of the pagination links.

## Environment

* MacOS
* Safari
* VoiceOver

## Possible Fixes (optional)

I reformatted the container announcing number of results to announce to assistive devices. I also added another span to read out the word "through" to assistive devices instead of "dash". Finally, I suggest we show the set of results like `1-10 of 162`, or `11-20 of 162`, instead of `10 of 162` as this is more meaningful.

```html
<!-- Added tabindex in https://github.com/department-of-veterans-affairs/va.gov-team/issues/981 -->
<div
  aria-live="polite"
  class="break-div"
  role="status"
  tabindex="-1"
>
  <!--
    UPDATE: Removing role="text" because it causes an aXe violation.
    Adding a role="text" to prevent iOS Safari from splitting the text.
    See https://axesslab.com/text-splitting/ for more details
  -->
  <span>
    Showing
    <span aria-hidden="true" id="total-pages">1 - 10</span>
    <span class="usa-sr-only" id="total-pages">1 through 10</span>
    <span id="total-all"> of 176</span>
    results
  </span>
</div>
```

## WCAG or Vendor Guidance (optional)
* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2019-08-14 at 2.10.10 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/c5a4b95f-61bc-40b4-942b-50dbc55e3e81)

**Comment by jefflbrauer** [comment]
@ethanteague See comments above from @1Copenut

**Comment by jefflbrauer** [comment]
@1Copenut pls review this again.

**Comment by 1Copenut** [comment]
@jefflbrauer This issue looks good, and is announcing changes to all three assistive devices I tested. I'm going to close it, however, keeping #981 open for another review. Please see my comment https://github.com/department-of-veterans-affairs/va.gov-team/issues/981#issuecomment-528606078 for context.
